@Mytherin 

For clang-tidy failure
```
/home/runner/work/duckdb/duckdb/src/function/pragma/pragma_queries.cpp:23:12: error: the 'empty' method should be used to check for emptiness instead of comparing to an empty object [readability-container-size-empty,-warnings-as-errors]
        schema = (schema == INVALID_SCHEMA) ? DEFAULT_SCHEMA : schema;
                  ^~~~~~~~~~~~~~~~~~~~~~~~
                  schema.empty()                 
``` 

Would you prefer I try to opt the line out of the clang-tidy rule or check for empty schema explicitly?

--- a/src/function/pragma/pragma_queries.cpp
+++ b/src/function/pragma/pragma_queries.cpp
@@ -20,7 +20,7 @@ string PragmaTableInfo(ClientContext &context, const FunctionParameters &paramet
 string PragmaShowTables(ClientContext &context, const FunctionParameters &parameters) {
 	auto catalog = DatabaseManager::GetDefaultDatabase(context);
 	auto schema = ClientData::Get(context).catalog_search_path->GetDefault().schema;
-	schema = (schema == INVALID_SCHEMA) ? DEFAULT_SCHEMA : schema;
+	schema = (schema == INVALID_SCHEMA) ? DEFAULT_SCHEMA : schema; // NOLINT
 
 	auto where_clause =
 	    StringUtil::Join({"where database_name = '", catalog, "' and schema_name = '", schema, "'"}, "");

--- a/test/sql/attach/attach_show_table.test
+++ b/test/sql/attach/attach_show_table.test
@@ -2,6 +2,8 @@
 # description: Show table should respect current scope
 # group: [attach]
 
+require skip_reload
+
 statement ok
 PRAGMA enable_verification
 

--- a/test/sql/pragma/test_show_tables.test
+++ b/test/sql/pragma/test_show_tables.test
@@ -2,8 +2,6 @@
 # description: Test SHOW/DESCRIBE tables
 # group: [pragma]
 
-require skip_reload
-
 statement ok
 CREATE TABLE integers(i INTEGER, j INTEGER)
 

